stl/inc/coroutine

```diff
@@ -145,9 +177,13 @@ struct noop_coroutine_promise {};
 
 // STRUCT coroutine_handle<noop_coroutine_promise>
 template <>
-struct coroutine_handle<noop_coroutine_promise> : coroutine_handle<> {
+struct coroutine_handle<noop_coroutine_promise> {

```

Comment:
LWG-3460 depicts `coroutine_handle<noop_coroutine_promise>` as having:

```cpp
// [coroutine.handle.noop.address], address
constexpr void* address() const noexcept;
```

but that isn't present here. This was found by the MSVC-internal test `src\qa\VC\FE\compiler\tests\devtest\await\noop\noop.cpp`.

Reply:
Fixed. I also added a bit of local test coverage for `noop_coroutine_handle` which would have caught this.